### ISSUE TITLE
Use of a reference translation

### ISSUE THREAD

**Issue by jeblad** [vslavik/poedit#749]
It could be nice to be able to use one or more reference translations as a baseline for further translation work.

For example while making a translation for Norwegian Nynorsk (nn) the Norwegian Bokmål (nb) translation could work as a reference translation. Those two forms (variants) of Norwegian is so close that it is way faster to fix strings from the other form than writing the string from scratch.

(Norwegian (no) is the macro code, but often abused for the Norwegian Bokmål (nb) variant, to much grief for people using the Norwegian Nynorsk (nn) variant.)

Given the present GUI of poedit, I guess would add such reference translations in the right column under "Proposals", but mark them with the original language and variant code.

**Comment by vslavik** [comment]
Mostly-duplicate of #449, #688.

The suggestion to put this sort of reference in sidebar is an interesting one, thanks.

**Comment by jeblad** [comment]
Yeah, I looked at those two, but found the proposals limited and a bit off.

**Comment by vslavik** [comment]
@jeblad Please don't spam the bugtracker with irrelevant, argumentative or merely talkative comments. This is not a conversation forum, but an issue tracker - a tool to keep track of issues and to record *relevant* information for handling them. Comments that don't add substantial information are just distracting noise.

**Comment by vslavik** [comment]
> The idea of loading a reference language is amazing but I think that an additional left sub pane would lead to a more effective workflow when translating (compared to the sidebar alternative). Thought shown in red below.

@bravo-kernel For the use case you show, where the source text is just symbolic ID and not proper text, both are just stop-gap measues (and the *real* good solution being to just have "Source text" replaced with the reference and show the ID as secondary information).

But the UI you propose is indeed better for the use case that @jeblad describes, i.e. when you need a similar language for reference. I like that.

>  (except poeditor .com)

Let's not link that name-squatter here...

**Comment by bravo-kernel** [comment]
Thanks for responding and considering 👍

_ps: you are correct, I have since dropped using IDs and have switched to natural language. Much better workflow._

**Comment by bravo-kernel** [comment]
@vslavik awesome, much appreciated ♥
